### PR TITLE
Document unsupported console observability features

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-7072

The following SQL O11Y features are not yet surfaced directly in the CC console until we have proper FGAC:

Statement diagnostic bundles (Statements page)
Dropping unused indexes (Insights and Databases pages)
Creating missing indexes (Insights)
Replacing existing indexes (Insights)

Also removed the Jobs page from unsupported features because it does exist in the Cloud Console and added a few links.

### CODE REVIEW

File: cockroachcloud/serverless-unsupported-features.md [hunk 1]
```diff
@@ -41,14 +41,19 @@ The [`ALTER TABLE ... SPLIT AT`](../{{site.current_cloud_version}}/alter-table.h
...
 - Hot ranges
-- Jobs
 - Advanced Debug
 
+The Cloud Console also does not currently provide the following features available in the DB Console:
+
+- [Statement diagnostic bundles](../{{site.versions["stable"]}}/ui-statements-page.html#diagnostics) on the **Statements** Page
+- [Dropping unused indexes](../{{site.versions["stable"]}}/ui-databases-page.html#index-recommendations) on the **Insights** and **Databases** pages
+- [Creating missing indexes](../{{site.versions["stable"]}}/ui-insights-page.html#schema-insights-view) and [replacing existing indexes](../{{site.versions["stable"]}}/ui-insights-page.html#schema-insights-view) on the **Insights** page
+
```
Comment (kevin-v-ngo) [inline cockroachcloud/serverless-unsupported-features.md:56]: Can we clarify that recommendations for dropping unused indexes and creating missing indexes are available? Just not direct actions?

Perhaps just prepending "Direct actions"?

**Direct actions** to drop unused indexes in the console.
**Direct actions** to create missing indexes and replace existing indexes in the console.

**Review by kevin-v-ngo — COMMENTED** [review]
Thanks for getting this out so quickly Gemma! Looks good. Small request to update.

File: cockroachcloud/serverless-unsupported-features.md (hunk 1 above)
Comment (gemma-shay) [reply cockroachcloud/serverless-unsupported-features.md:56]: sounds good!

**Review by rmloveland — APPROVED** [review]
LGTM!

**Pull request merged**